setup-azad/setup-azad-auth_provider.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+# This create an AzAD authentication provider in Canvas 
+# typically this is used for Canvas Beta which will have a URL of /login/saml/139 but could be used on any server 
+
```

Comment:
I'd normally have a `set -e` so it fails on any error, is there a reason not to for this script?